developer-docs/dev_setup.md

```diff
@@ -1,215 +1,145 @@
 
-You can connect to the VPN using the profile downloaded at the previous step with:
+  - [Openvpn](#openvpn)
+  - [Docker](#install-docker)
+  - [kubectl](#install-kubectl)
+  - [Drone](#drone)
+  - [KD](#kd)
 
-```bash
-$ sudo openvpn --config <vpn_profile_file>
-Tue Dec  6 11:24:04 2016 Initialization Sequence Completed
-```
-
-The profile expires after 12 hours. You'll need to download and connect to a new VPN Profile when it expires.
-
-## Install Docker
-
-You can follow the instructions to install `docker` from the [Docker website](https://docs.docker.com/engine/installation/). You can verify the installation is successful with:
+#### Openvpn
 
-```bash
-$ docker --version
-Docker version 1.12.3, build 6b644ec
-```
+Most of ACP's services operate behind a VPN.
 
-## Install and configure kubectl
+You can connect to the vpn using `openvpn`.  You can find instructions on how to install and operate `openvpn` for your OS: [OpenVPN].
 
-### Install kubectl
+>Note:  All examples in this document were made with a Linux OS - other operating systems may vary
 
-You can follow the instructions to install `kubectl` from the [Kubernetes website](http://kubernetes.io/docs/user-guide/prereqs/). You can verify the installation is successful with:
+You can verify the installation was successful with:
 
 ```bash
-$ kubectl version
-Client Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.6+e569a27", GitCommit:"e569a27d02001e343cb68086bc06d47804f62af6", GitTreeState:"not a git tree", BuildDate:"2016-11-12T09:26:56Z", GoVersion:"go1.7.3", Compiler:"gc", Platform:"darwin/amd64"}
-Server Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.5", GitCommit:"5a0a696437ad35c133c0c8493f7e9d22b0f9b81b", GitTreeState:"clean", BuildDate:"1970-01-01T00:00:00Z", GoVersion:"go1.7.1", Compiler:"gc", Platform:"linux/amd64"}
-```
-
-### Configure kubectl
-
-You will need to configure the `kubectl` client with the appropriate details to be able to talk to the kubernetes cluster.
-We recommend copying the [example kubectl config](resources/kubeconfig) to `~/.kube/config`. Note that you may need to create this directory and file.
-
-The only change you will need to make is to replace _"XXXXXXXXXX"_ with your kubernetes token.
-
-You can submit a request to obtain a kubernetes token by adding an issue to the Hosting Platform Bau project on Github [here](https://github.com/UKHomeOffice/application-container-platform-bau/issues).
-
-Please use the below template for your request:
-
-```
-Please can I have a kubernetes token with access to my teams namespaces.  
-Email: xxx.xxx@digital.homeoffice.gov.uk  
-Name: xxx xxx  
-Team: My Project Team  
-Public GPG Key: xxxxxxxxx
-Namespace: dev-induction
+$ openvpn --version
+OpenVPN 2.3.13 x86_64-apple-darwin16.1.0 [SSL (OpenSSL)] [LZO] [MH] [IPv6] built on Nov  5 2016
```

Comment:
This doesn't mean much if they have routing or nameserver issues. Please ask them to refer to the section below where they download an ACP profile and then resolve the Platform Hub.